src/query.rs

```diff
@@ -65,6 +65,10 @@ pub unsafe trait Fetch<'a>: Sized {
     unsafe fn get(&self, n: usize) -> Self::Item;
 }
 
+/// Marker trait indicating whether a given [`Fetch`] implementation will produce only shared references
```

Comment:
"will not produce unique references" might be a little more accurate, since e.g. `Satisfies` produces non-reference data.

Reply:
Changed as suggested.